libsodium-wrappers 0.7.16 doesn't resolve libsodium.mjs

If you build with the newest version of `libsodium-wrappers`, which is `0.7.16` it won't resolve libsodium.mjs:

```txt
Error: Cannot find module '/tmp/forum/node_modules/.pnpm/libsodium-wrappers@0.7.16/node_modules/libsodium-wrappers/dist/modules-esm/libsodium.mjs' imported from /tmp/forum/node_modules/.pnpm/libsodium-wrappers@0.7.16/node_modules/libsodium-wrappers/dist/modules-esm/libsodium-wrappers.mjs
```

To fix this is enough to bump to `0.8`.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,11 +11,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## 2026-01-08: v0.20.2
 
-### Added
 ### Fixed
 - Bump `libsodium` to `0.8` to fix `libsodium.mjs` resolution
-### Changed
-### Removed
 
 ## 2025-12-04: v0.20.1
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 ### Removed
 
+## 2026-01-08: v0.20.2
+
+### Added
+### Fixed
+- Bump `libsodium` to `0.8` to fix `libsodium.mjs` resolution
+### Changed
+### Removed
+
 ## 2025-12-04: v0.20.1
 
 ### Added

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@holochain/client",
-  "version": "0.20.1",
+  "version": "0.20.2",
   "description": "A JavaScript client for the Holochain Conductor API",
   "author": "Holochain Foundation <info@holochain.org> (https://holochain.org)",
   "license": "CAL-1.0",
@@ -48,7 +48,7 @@
     "isomorphic-ws": "^5.0.0",
     "js-base64": "^3.7.8",
     "js-sha512": "^0.9.0",
-    "libsodium-wrappers": "^0.7.15",
+    "libsodium-wrappers": "^0.8",
     "lodash-es": "^4.17.21",
     "ws": "^8.18.3"
   },

